(Bagel) Mapping Catch-up

CawsForConcern opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
This is a general mapping feature pass.

Specific changes to this map:
- [x] Added turrets to AI Core
- [x] Added Turret Control Panel to AI Core
- [x] Moved the material silo in Science so it can reach Robotics
- [x] Moved the station beacon for Robotics into Robotics (it wasn't in Robotics before)

Feature-catch up:
- [x] Nanomed Civilians mapped from #2092
- [x] AI Lawboard crate from #1507
- [x] Surgery & Robotics doors from #1520
- [x] HV & Pipe dock support for #1727
- [x] New furniture from #2050

QoL:
- [x] Ensure HOP has a document printer mapped
- [x] Ensure Medical has a document printer mapped
- [x] Ensure Library has a document printer and fax mapped
- [x] Ensure Chemistry has Cell Recharger mapped
- [x] Ensure Brigmedic has Cell Recharger mapped

Bugfixes:
- [x] Correct EVA Doors from #1741
- [x] Double check that Lawyer + SEC airlocks are properly used (for IAA, Magistrate)

## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
- Nanomed Vending machines were asked to be mapped
- Implementing the AI Lawboard crate makes it so that maps don't need to be updated every time we change the lawboards
- Surgery and Robotics airlocks were added a while ago, this map pass just ensures they're actually being used
- EVA doors are wrong on some maps (some maps use HOP airlocks on EVA), and using EVA doors makes it easier to adjust accesses without updating maps
- The new furniture looks cool and gives Starlight maps some more visual differences from other forks
- HV cables (and pipes at Cargo Docks) need to be added across all maps to support shuttles that will use these features

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->
<img width="591" height="408" alt="image" src="https://github.com/user-attachments/assets/e097f462-e2d6-461b-8ee3-c1f8b0cc30ba" />

fig. 1 - Nanomeds mapped

<img width="846" height="677" alt="image" src="https://github.com/user-attachments/assets/3a3bd3d2-7a8a-4eba-af07-52e2218d3a97" />

fig. 2 - AI lawboards crate

<img width="633" height="433" alt="image" src="https://github.com/user-attachments/assets/e9c3c153-2fe1-4703-b18f-32cc9479c5f2" />

<img width="787" height="609" alt="image" src="https://github.com/user-attachments/assets/5f1910eb-89e0-45b8-b1da-0603ed02c64a" />

fig. 3 - Surgery and Robotics airlocks mapped
<img width="1191" height="745" alt="image" src="https://github.com/user-attachments/assets/38428aea-b887-4ccf-a0bf-cc83fcff5fa2" />

fig. 4 - HV cables (and pipes for Cargo only) mapped to all docks

<img width="1098" height="633" alt="image" src="https://github.com/user-attachments/assets/f3e3fa78-84cd-4bf5-9b63-89ac29cf1577" />
<img width="835" height="533" alt="image" src="https://github.com/user-attachments/assets/10fc6aa7-26cc-4cbd-8b29-1d6e72ea87bc" />
<img width="1008" height="996" alt="image" src="https://github.com/user-attachments/assets/58d56ac3-da6a-4930-871f-9608103e204b" />

fig. 5 - Some examples of new furniture mapped (not exhaustive, but generally went around and replaced old furniture with couches / benches / etc. where it would make sense and look good. Mostly Bars, Bridges, Offices, Evac, Arrivals.

<img width="675" height="556" alt="image" src="https://github.com/user-attachments/assets/222ada76-3a20-4711-824e-b16e8b7e79b2" />

fig. 6 - EVA Airlock door is correct

<img width="601" height="646" alt="image" src="https://github.com/user-attachments/assets/aa1fd579-2c56-4773-a4b4-362408df4daf" />
<img width="590" height="428" alt="image" src="https://github.com/user-attachments/assets/6888abdc-56a4-4279-a149-8f8a4bcac1c0" />
<img width="657" height="702" alt="image" src="https://github.com/user-attachments/assets/ff2d1484-b544-4e90-b4b0-f59f5a8e7f65" />

fig. 7 - Document printers at HOP, Medical, Library

<img width="1081" height="554" alt="image" src="https://github.com/user-attachments/assets/cd3fe683-82df-49f4-b7af-58ebcfbbad51" />
<img width="878" height="792" alt="image" src="https://github.com/user-attachments/assets/3d27312b-b801-40b1-9e0f-083102f9329a" />

fig. 8 - Brig doors for most access into SEC, SEC/Lawyer doors on Interrogation Room. Generally, someone with lawyer access should always be able to get to GenPop and the Interrogation Room

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [ ] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: STARLIGHT TEAM
- add: Added Starlight.
- remove: Removed SS13.
- tweak: Changed SS14.
- fix: Fixed Rinary.
-->
:cl: CawsForConcern
- add: (Bagel) New furniture! Updated benches, chairs, sofas added at select locations
- add: (Bagel) Nanomed civilian machines
- add: (Bagel) AI Lawboard crate (replaces old loose lawboards)
- add: (Bagel) Surgery access doors in Medical
- add: (Bagel) Robotics access doors in Science
- add: (Bagel) HV cables to shuttle docks
- add: (Bagel) Gas pipes to cargo shuttle docks
- add: (Bagel) Document printer to HOP
- add: (Bagel) Document printer to Medical
- add: (Bagel) Document printer to Library
- add: (Bagel) Cell Recharger to Chemistry
- add: (Bagel) Cell Recharger to Brigmedic
- add: (Bagel) Turrets to AI Core
- add: (Bagel) Turrets Control Panel to AI Core
- tweak: (Bagel) Moved Science's Material Silo so it can reach Robotic's Exosuit Fabricator
- tweak: (Bagel) Moved the Robotics Beacon into Robotics (it wasn't in Robotics before)
- fix: (Bagel) Fixed wrong door on EVA Storage (was HOP door)
- fix: (Bagel) SEC doors updated with Lawyer SEC doors where appropriate

CawsForConcern committed:
- add: (Bagel) New furniture! Updated benches, chairs, sofas added at select locations
- add: (Bagel) Nanomed civilian machines
- add: (Bagel) AI Lawboard crate (replaces old loose lawboards)
- add: (Bagel) Surgery access doors in Medical
- add: (Bagel) Robotics access doors in Science
- add: (Bagel) HV cables to shuttle docks
- add: (Bagel) Gas pipes to cargo shuttle docks
- fix: (Bagel) Fixed wrong door on EVA Storage (was HOP door)

CawsForConcern committed:
- add: (Bagel) Document printer to HOP
- add: (Bagel) Document printer to Medical
- add: (Bagel) Document printer to Library

CawsForConcern committed:
god damn it (fixes Lawyer / Brig access *again*)

CawsForConcern committed:
- add: (Bagel) Cell Recharger to Chemistry
- add: (Bagel) Cell Recharger to Brigmedic

CawsForConcern committed:
- add: (Barratry) Cell Recharger to Chemistry
- add: (Barratry) Cell Recharger to Brigmedic

CawsForConcern committed:
Merge branch 'MapPassBagel' of github.com:CawsForConcern/space-station-14 into MapPassBagel

R3v3l4t1on commented:
I forget but I seem to recall that bagel has no turrets. Might be me getting confused with downstreams though. Still, can this be checked? I also recall that bagel's silo (if it even has one roundstart) cannot reach the exofab.

CawsForConcern commented:
> I forget but I seem to recall that bagel has no turrets. Might be me getting confused with downstreams though. Still, can this be checked? I also recall that bagel's silo (if it even has one roundstart) cannot reach the exofab.

Added Turrets to AI along with a Turret Control Panel.
<img width="1340" height="998" alt="image" src="https://github.com/user-attachments/assets/91936281-852d-4ec2-b05a-efd450aa3e94" />


Moved Silo so it can reach all of Science's machines.
<img width="1204" height="621" alt="image" src="https://github.com/user-attachments/assets/02262f71-b47e-4e68-a132-fd5035292c24" />

Changelog updated

CawsForConcern committed:
- add: (Bagel) Turrets to AI Core
- add: (Bagel) Turrets Control Panel to AI Core
- tweak: (Bagel) Moved Science's Material Silo so it can reach Robotic's Exosuit Fabricator
- tweak: (Bagel) Moved the Robotics Beacon into Robotics (it wasn't in Robotics before)

CawsForConcern committed:
Merge branch 'starlight-dev' into MapPassBagel

CawsForConcern committed:
new pews now that they're fixed yay

Happyrobot33 committed:
Merge branch 'starlight-dev' into MapPassBagel
